one-app-statics/package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "@americanexpress/one-app-statics",
+  "version": "5.0.0-rc1",
+  "description": "One App Static Assets",
+  "main": "index.js",
+  "scripts": {
```

Comment:
Can you add this section to the package.json it would allow us to be able to guide contributors to this repo from npm
 `  "homepage": "https://github.com/americanexpress/<name-of-project>",
  "bugs": {
    "url": "https://github.com/americanexpress/<name-of-project>/issues"
  } `